### PR TITLE
New demo: service-spawner

### PR DESCRIPTION
**Pull request by jotak**

### CODE REVIEW

File: service-spawner/gentpl.sh [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
```
Comment (jmazzitelli) [inline service-spawner/gentpl.sh:1]: What does "gentpl" stand for? Can we rename this to something more human-readable/understandable? It isn't a linux utility, so we are free to use more than 6 letters in a command name :) How about "generate-circular-services.sh" or something descriptive like that?

**Review by jmazzitelli — CHANGES_REQUESTED** [review]

File: service-spawner/gentpl.sh (hunk 1 above)
Comment (jotak) [reply service-spawner/gentpl.sh]: I don't like very long names, how about a compromise : gen-services.sh ? :-)

**Review by jmazzitelli — APPROVED** [review]
lgtm

**Comment by lucasponce** [comment]
Thanks @jotak !

**Pull request merged**